CoC.md

```diff
@@ -3,10 +3,12 @@
 ## Preamble
 
 Like the broader technical and activism communities as a whole, the SecureDrop
-team is made up of a mixture of staff and volunteers from all over the world,
-working on every aspect of our mission—including operations, software
+Community is made up of a mixture of staff and volunteers from all over the world,
+engaged in various activities—including operations, software
 development, mentorship, and building connections with great people and
 organizations.
+All individuals who associate themselves with the SecureDrop Community are on equal footing, regardless of their affiliation with any specific organization.
```

Comment:
Hey @dachary could we remove lines 10-11 here just so we can get this merged in? We're very nearly done with the CoC and I'd rather have something finished and then continue the debate on how to flatten the organizational structure and decentralize further. I still feel like this statement is a little outside the scope of the CoC, and I agree with @eloquence that it's still not 100% true in the sense that the forum has mods and the Community Team has some authority not granted to others (thus there is some hierarchy).